x-pack/package.json

```
@@ -213,6 +213,7 @@
     "bluebird": "3.5.5",
     "boom": "^7.2.0",
     "brace": "0.11.1",
+    "broadcast-channel": "^3.0.2",
```

Optional -- looks like we just had a version bump
```suggestion
    "broadcast-channel": "^3.0.3",
```